test/functional/cmdLineTests/jvmtitests/src/com/ibm/jvmti/tests/samplingObjectAllocation/soae001.java

```diff
@@ -46,16 +46,22 @@ public boolean testDefaultInterval() {
 			if (samplingResult < 1) {
 				System.out.println("com.ibm.jvmti.tests.samplingObjectAllocation.soae001.check() failed, expected 1+ but got: " + samplingResult);
 			} else {
+				jvmtiResult = enable(Thread.currentThread());
+				if (0 != jvmtiResult) {
+					System.out.println("com.ibm.jvmti.tests.samplingObjectAllocation.soae001.enable(thread) failed as expected with: " + jvmtiResult);
+				}
+				
 				jvmtiResult = disable();
 				if (0 != jvmtiResult) {
 					System.out.println("com.ibm.jvmti.tests.samplingObjectAllocation.soae001.disable() failed with: " + jvmtiResult);
 				} else {
 					reset();
-					bytes = new byte[DEFAULT_SAMPLING_RATE];
+					bytes = new byte[DEFAULT_SAMPLING_RATE * 10];
```

Comment:
Why times 10?

Reply:
Current sampling rate might not be exact as the sampling interval specified as per https://github.com/eclipse/openj9/issues/6024.
Increasing the allocation size in hope to avoid false pass due to insufficient sampling.

Reply:
Would it not make more sense to just not write off the end of the array in that case?

Reply:
Discussed with @gacholio , the test for no sampling event fired after `disable()` is not needed considering the nature of non-deterministic event generation.
Removed the test in question, please have another look.